src/logic/operate.js

```diff
@@ -11,13 +11,11 @@ export default function operate(numbernumber1, numbernumber2, operation) {
     case 'x':
       return number1.times(number2).toString();
     case '/':
-      if (number2 === 0) {
+      if (numberTwo === 0)
         alert(`Division by Zero unauthorized`);
-        return;
-      }
-      else {
+      else
         return number1.div(number2).toString();
-      }
+      break;
```

Comment:
sorry 😭 but this is still going to break. the button name is a string and using `===` checks for type also so comparing string to integer will fail.
![Screenshot 2019-10-24 at 8 32 58 AM](https://user-images.githubusercontent.com/10922508/67463241-f6291600-f638-11e9-83e8-5099b13b04ba.png)



- please fix this.

Reply:
Thank you for this reminder '==='